packages/config/rollup.config.js

```diff
@@ -0,0 +1,58 @@
+import typescript from 'rollup-plugin-typescript2';
```

Comment:
Shared rollup config file. Currently there is only one module (`cbioportal-frontend-commons`) using this config. But the plan is to move `react-mutation-mapper` under this monorepo as well.